python/remote_api/src/lynx/kite.py

```diff
@@ -884,6 +884,40 @@ def f(**kwargs):
     _add_documentation_to_operation(f, name)
     return f
 
+  @property
+  def vertices(self) -> 'SingleOutputAtomicBox':
+    '''Returns the vertices as a table.'''
+    return self._select_all('vertices')
+
+  @property
+  def edges(self) -> 'SingleOutputAtomicBox':
+    '''Returns the edges as a table.'''
+    return self._select_all('edges')
+
+  @property
+  def graph_attributes(self) -> 'SingleOutputAtomicBox':
+    '''Returns the graph attributes as a table.'''
+    return self._select_all('graph_attributes')
+
+  @property
+  def edge_attributes(self) -> 'SingleOutputAtomicBox':
+    '''Returns the edge attributes as a table.'''
+    return self._select_all('edge_attributes')
+
+  @property
+  def segmentations(self):
+    '''Access a given segmentation.
+
+    Example usage:
+    ```
+    graph = lk.createExampleGraph().findConnectedComponents(name='seg1')
+    segmentation = graph.segmentations['seg1']
```

Comment:
I think this syntax makes it look like `graph.segmentations` is a dict. So a user may try `for s in graph.segmentations` or stuff like that which won't work. For this reason I think a simple function call is more honest and not any more verbose.

```suggestion
    segmentation = graph.segmentation('seg1')
```

Reply:
Done. Is this approximately what you had in mind?